k8s/demo/common/divorce/div-cos.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: div-cos
+  namespace: divorce
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: div-cos
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: div-cos
+    version: 1.3.2
+  values:
+    java:
+      replicas: 2
+      memoryRequests: '768Mi'
```

Comment:
you shouldn't need these memory limited and cpu requests

Reply:
Why? Are these the default values?
Should I remove it only from the demo file or from all of them?

Reply:
tbh, not sure the history, 95% of apps don't need it, it was probably copied from the original, but might be too much work to find out

Reply:
fair enough. I'll leave it, in that case.